scripts/prepare_automatic_backups.py

```diff
@@ -58,10 +58,10 @@ def update_cron_dict(cron_dict):
 
     Args:
         cron_dict: dict(str ,str). Content of yaml file in dictionary type.
```

Comment:
Also, add `The keys and values are as follows:` after this line.

Reply:
resolved it